osu.Game.Rulesets.Tau/Objects/Drawables/Pieces/HardBeatPiece.cs

```diff
@@ -30,6 +33,13 @@ public HardBeatPiece()
             };
 
             NoteSize.BindValueChanged(value => BorderThickness = convertNoteSizeToThickness(value.NewValue));
+            HighlightHardBeats.BindValueChanged(value => BorderColour = value.NewValue ? Color4.Orange : Color4.White);
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(TauRulesetConfigManager config)
```

Comment:
Prolly better to mark this as nullable (`TauRulesetConfigManager?`) since you treat is as nullable underneath.